### PR TITLE
Adding Bala Hantsi's profile

### PR DESCRIPTION
**Pull request by bhantsi**

### CODE REVIEW

**Comment by bhantsi** [comment]
Add my profile

File: public/data/bala-hantsi.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "color": "grey",
+      "description": "Started freelancing"
+    },
+    {
+      "title": "Started Mentoring",
+      "date": "March 2019",
+      "description": "Started mentoring"
+    },
+    {
+      "title": "Started engaigin in developers community",
```
Comment (Panquesito7) [inline public/data/bala-hantsi.json:32]: Spelling error here; might want to fix. 🙂
```suggestion
      "title": "Started engaging in developers' community",
```

File: public/data/bala-hantsi.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/bhantsi.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/bhantsi",
+      "icon": "github"
+    },
+    {
+      "name": "Bala Hantsi",
```
Comment (syedsohan) [inline public/data/bala-hantsi.json:13]:
```suggestion
      "name": "Follow me on Twitter",
```

File: public/data/bala-hantsi.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      "description": "Started freelancing"
+    },
+    {
+      "title": "Started Mentoring",
+      "date": "March 2019",
+      "description": "Started mentoring"
+    },
+    {
+      "title": "Started engaigin in developers community",
+      "date": "Feb 2020",
```
Comment (syedsohan) [inline public/data/bala-hantsi.json:33]:
```suggestion
      "date": "February 2020",
```

File: public/data/bala-hantsi.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      "description": "Started leading OSACA KANO (an open source community Africa local chapter)"
```
Comment (syedsohan) [inline public/data/bala-hantsi.json:34]:
```suggestion
      "description": "Started leading OSACA KANO (an open-source community Africa local chapter)"
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**